client/package.json

```diff
@@ -97,7 +97,7 @@
 		"autosize": "^4.0.2",
 		"body-parser": "^1.19.0",
 		"browser-filesaver": "^1.1.1",
-		"browserslist-useragent": "^3.0.3",
+		"browserslist-useragent": "browserslist/browserslist-useragent#64/head",
```

Comment:
Fixes electron compatibility: https://github.com/browserslist/browserslist-useragent/pull/64

Using to test this PR; not sure if we'll want to merge as is.

Reply:
Going on a tangent here, but I had no idea you could install packages from PRs 🤯 TIL